kiwi/utils/command_capabilities.py

```diff
@@ -50,10 +50,11 @@ def has_option_in_help(
 
         :rtype: bool
         """
+        help_args = help_flags or ['--help']
```

Comment:
Just out of curiosity, what is the smelling part of having default values different than `None`? I am perfectly fine with this modified code, just want to understand the landscape reasoning.

Reply:
See this: https://www.pythoncentral.io/fun-with-python-function-parameters/
[Potential Python Function Parameter Problems]

Reply:
Wow, interesting reading... That's valuable knowledge, thanks!